### DISCUSSION TITLE
l4t r35.2.1 Integration

### DISCUSSION THREAD

**Discussion by elbbit01** [OE4T/meta-tegra#1163] [General]
Hello,
I dont intend to put pressure on you since this branch is so helpful, but for planning reasons I would like to know if that wip branch will be adapted also for Kirkstone and more or less how much do you estimate. 
We are using the kirkstone branch at the moment and really hanging on the UEFI AB partitioning handling and boot tools, and it is supposed to work on that l4t release. 
Regards,
Alvaro.

**Comment by madisongh** [comment]
Yes, the JP 5.1/L4T R35.2.1 upgrade will be applied to kirkstone as well.  I don't think I've seen any real improvements in A/B support or the bootloader update tooling, though.

**Reply by elbbit01** [reply]
Hi madisongh,
I got the information from nvidia, that nvidiabootctl functions that were still not working in the previous jetpack, like "set-good" a partition, are working now. It is also expected from the release notes, if I checked it correctly. 
I speak about Jetson Xavier AGX.
Thanks in advance,
Alvaro.
